Apsitvarkom.DataAccess/PollutedLocationDatabaseRepository.cs

```diff
@@ -42,6 +42,14 @@ orderby pollutedLocation.Location.Coordinates.DistanceTo(inRelationTo)
             .FirstOrDefaultAsync(propertyCondition);
     }
 
+    /// <inheritdoc />
+    public async Task<bool> ExistsByPropertyAsync(Expression<Func<PollutedLocation, bool>> propertyCondition)
+    {
+        return await _context.PollutedLocations
+            .Include(l => l.Events)
+            .AnyAsync(propertyCondition);
+    }
```

Comment:
Possible to return `Task` directly?

Reply:
See https://github.com/vu-vibedosa/apsitvarkom-be/pull/114#discussion_r1033876775

Reply:
I guess I will resolve all of these in #114. It would unnecessarily include `CleaningEventDatabaseRepository` into this as well [as the same is replicated there], which is kind of unrelated to this task (even though I technically changed the files due to renaming).